public/controllers/management/components/management/reporting/reporting-table.js

```diff
@@ -81,11 +94,31 @@ class WzReportingTable extends Component {
     const columns = this.reportingColumns.columns;
     const message = isLoading ? null : 'No results...';
 
+    const deleteReport = (itemList) => {
+      try {
+        this.deleteReport(itemList);
```

Comment:
**issue**: this method is asynchronous, should we wait until the execution is completed?